srcpkgs/cookie/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'cookie'
+pkgname=cookie
+version=0.1.1
+revision=1
+_commit=9f9b6eedee297eb9148f0a5e3d9993a8b15b9f66
+create_wrksrc=yes
+build_wrksrc="${pkgname}-${version}"
+short_desc="Template-based File Generator"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://github.com/bbugyi200/cookie"
+distfiles="${homepage}/archive/v${version}.tar.gz
+ https://github.com/bbugyi200/bashlibs/archive/${_commit}.tar.gz"
```

Comment:
shouldn't it be a separate package that cookie depends on ?

Reply:
Can do it it, yes. Though only consists of two script files and a Makefile.